### PR TITLE
Created the escape menu and removed topMenu

### PR DESCRIPTION
**Pull request by jamzam90**
Buttons from the Topmenu gameobject moved to escape menu as well as the dropdown for langs.

### CODE REVIEW

**Comment by jamzam90** [comment]
The keycode ifor the escape is set to the spacebar as I was testing multikey keybinds and ctrl and esc for some odd reason on windows 10 opens the start menu.

**Comment by WardBenjamin** [comment]
This looks good, but I think I like the work done in #376 more. @RodRitter @vogonistic opinions?

**Comment by RodRitter** [comment]
Agreed. Not that this is bad, it's just that #376 is more fully featured.

File: Assets/Scripts/Controllers/UIController.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+public class UIController : MonoBehaviour {
```
Comment (OzieGamma) [inline Assets/Scripts/Controllers/UIController.cs]: Braces should go on the next line

File: Assets/Scripts/Controllers/UIController.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	public WorldController wCont;
```
Comment (OzieGamma) [inline Assets/Scripts/Controllers/UIController.cs]: I had trouble understanding what wCont was when I first read the code. I suggest changing it to worldController to make the code more readable.

**Comment by WardBenjamin** [comment]
Closing due to #376 being merged, but if you have any improvements that you think would be good, feel free to open a new PR!

**Pull request closed without merging**